build.xml

```diff
@@ -37,7 +37,7 @@ xmlns:cs="antlib:com.puppycrawl.tools.checkstyle.ant">
 
     <property name="audience-annotations.version" value="0.5.0" />
 
-    <property name="netty.version" value="4.1.29.Final"/>
+    <property name="netty.version" value="4.1.32.Final"/>
```

Comment:
Please update Maven pom as well.

What about moving to latest netty?
I guess we should update boring ssl dependency as well

Reply:
Good catch! I'll fix up the pom.xml and see about bumping it to the latest netty: 4.1.36.